ui/src/model/ModelDetails.js

```diff
@@ -34,34 +30,23 @@ const LoadingContent = () => (
 
 export const ModelDetails = () => {
   const { projectId, modelId } = useParams();
-  const location = useLocation();
-  const [model, setModel] = useState(get(location.state, "model"));
-  const [breadcrumbs, setBreadcrumbs] = useState([]);
 
-  const [{ data: models, isLoaded: modelsLoaded }] = useMerlinApi(
-    `/projects/${projectId}/models`,
-    {},
-    [],
-    !model
+  const [{ data: model, isLoaded: modelLoaded }] = useMerlinApi(
+    `/projects/${projectId}/models/${modelId}`,
+    { mock: mocks.model },
+    []
   );
 
-  useEffect(() => {
-    modelsLoaded && setModel(models.find(m => m.id.toString() === modelId));
-  }, [models, modelsLoaded, modelId, setModel]);
-
-  useEffect(() => {
-    model &&
-      setBreadcrumbs([
-        {
-          text: "Models",
-          href: `/merlin/projects/${projectId}/models`
-        },
-        {
-          text: model.name,
-          href: `/merlin/projects/${projectId}/models/${model.id}`
-        }
-      ]);
-  }, [projectId, model]);
+  const breadcrumbs = [
+    {
+      text: "Models",
+      href: `/merlin/projects/${projectId}/models`,
+    },
+    {
+      text: model.name,
+      href: `/merlin/projects/${projectId}/models/${model.id}`,
```

Comment:
Though the component will be reloaded when the `model` value changes, shall we simply use the constant `modelId` here instead of `model.id`?

Reply:
You are right. Thanks for the suggestion. Updated.